tensorflow/python/kernel_tests/string_split_op_test.py

```diff
@@ -145,6 +145,56 @@ def testStringSplitWithNoSkipEmpty(self):
       self.assertAllEqual(indices, [[0, 0], [1, 0], [2, 0]])
       self.assertAllEqual(shape, [3, 1])
 
+  def testStringSplitWithUtf8AndEmptyDelimiter(self):
+    # utf8 \xE6\x82\xA8 \xE5\xA5\xBD, \xE6\x82\xA8 \xE5\xA5\xBD
+    strings = [b"\xE6\x82\xA8\xE5\xA5\xBD", b"\xE4\xB8\x96\xE7\x95\x8C"]
+
+    with self.test_session() as sess:
+      tokens = string_ops.string_split_utf8(strings, delimiter="")
```

Comment:
I don't see any tests for `skip_empty=True`. Also, it appears that `skip_empty` isn't handled. If you'd rather get rid of the `skip_empty` op attribute altogether, that's fine. If not, let's make it work :)

Reply:
Ping?

Reply:
@asimshankar The tests have been updated with test cases covered when skip_empty=False. Ln 160-208.